### PR TITLE
Cleaned up date and time validations and messages

### PR DESCRIPTION
**Pull request by IanCrisp**

### CODE REVIEW

File: DFC.App.ContactUs/wwwroot/js/apps/dfc-app-contactus.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿class DfcAppContactUs {
```
Comment (snipered2003) [inline DFC.App.ContactUs/wwwroot/js/apps/dfc-app-contactus.js:1]: Are we definitley using es6?

File: DFC.App.ContactUs/Views/EnterYourDetails/EnterYourDetailsView.cshtml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <form asp-route="/pages/enter-your-details" method="post">
+        <form asp-route="/pages/enter-your-details" method="post" class="compui-validation">
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/EnterYourDetails/EnterYourDetailsView.cshtml:15]: Yes this is newly added to digital-assets

File: DFC.App.ContactUs/Views/Home/HomeBody.cshtml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿@model HomeBodyViewModel
 
-<form asp-route="/contact-us/home" method="post">
+<form asp-route="/contact-us/home" method="post" class="compui-validation">
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/Home/HomeBody.cshtml:3]: Yes this is newly added to digital-assets

File: DFC.App.ContactUs/Views/Home/HomeForm.cshtml [hunk 4]
```diff
@@ -9,7 +9,7 @@
         </legend>
         <p>
             Call @Model.ServiceOpenDetailModel.PhoneNumber to speak to an adviser. Lines are open from  @Model.ServiceOpenDetailModel.OpenTimesString,  @Model.ServiceOpenDetailModel.OpenDays. Calls are free from landlines and most mobile numbers.
-            Find out more about <a href="https://www.gov.uk/call-charges">call charges and freephone numbers</a>
+            Find out more about <a href="https://www.gov.uk/call-charges">call charges and freephone numbers</a>.
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/Home/HomeForm.cshtml:12]: it is a fully stop for the sentence

File: DFC.App.ContactUs/Views/Home/HomeView.cshtml [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <form asp-route="/pages/home" method="post">
+        <form asp-route="/pages/home" method="post" class="compui-validation">
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/Home/HomeView.cshtml:16]: Yes this is newly added to digital-assets

File: DFC.App.ContactUs/Views/Shared/_LocalCss.cshtml [hunk 6]
```diff
@@ -4,3 +4,4 @@
 
 <link href="@cdnLocation/css/all.min.css" rel="stylesheet" type="text/css" />
 
+<link href="~/css/dfc-app-contactus.css" rel="stylesheet" type="text/css" asp-append-version="true" />
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/Shared/_LocalCss.cshtml:7]: Everything is in the CDN, this script is only used when running locally outside of the shell, allows all the scripts to be loaded and developed before submitting them to the CDN.
When the app runs inside the shell, this script is not used

File: DFC.App.ContactUs/Views/Shared/_LocalJs.cshtml [hunk 7]
```diff
@@ -8,4 +8,11 @@
 <script>window.GOVUKFrontend.initAll()</script>
 
 <script src="@cdnLocation/js/dfcdigital.min.js"></script>
-<script src="@cdnLocation/js/compui.min.js"></script>
+@*<script src="@cdnLocation/js/compui.min.js"></script>*@
+
+
+
+<script src="~/js/compui-utilities.js" asp-append-version="true"></script>
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/Shared/_LocalJs.cshtml:15]: These are just local copies that i was using to develop them. They are only used when running locally outside of the shell.
They are no in the CDN so used from there when running in the shell.

File: DFC.App.ContactUs/wwwroot/js/apps/dfc-app-contactus.js (hunk 1 above)
Comment (IanCrisp) [reply DFC.App.ContactUs/wwwroot/js/apps/dfc-app-contactus.js:1]: No restriction that I'm aware of

File: DFC.App.ContactUs/Views/WhyContactUs/WhyContactUsBody.cshtml [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ﻿@model WhyContactUsBodyViewModel
 
-<form asp-route="/contact-us/why-do-you-want-to-contact-us" method="post">
+<form asp-route="/contact-us/why-do-you-want-to-contact-us" method="post" class="compui-validation">
```
[the remark this answers is not in the document]
Comment (IanCrisp) [reply DFC.App.ContactUs/Views/WhyContactUs/WhyContactUsBody.cshtml:3]: Yes this is newly added to digital-assets - used a a selector class by script

**Comment by snipered2003** [comment]
Seems ok from the comments.

**Pull request merged**